src/webhooks/pubsub/slackScores.ts

```diff
@@ -198,17 +208,24 @@ export const sendGitHubEngagementMetrics = async (
   await Promise.all(slackNotifications);
 };
 
-export const sendDiscussionMetrics = async (ospo_internal: boolean = false) => {
-  const { discussions, discussionCommenters } = await getDiscussionEvents();
-  if (discussions.length === 0 && discussionCommenters.length === 0) {
+export const sendGitHubActivityMetrics = async (
+  ospo_internal: boolean = false
+) => {
+  const { discussions, githubCommenters, issues } =
+    await getGitHubActivityMetrics();
+  if (
+    discussions.length === 0 &&
```

Comment:
nit: factor this compound condition into a separate variable, makes the code much more readable. Maybe 
```suggestion
    const noGitHubActivityFound = (!discussions && !githubCommenters && !issues)
    if (noGitHubActivityFound) {
```